src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -104,9 +104,7 @@ const webhookRouter = async (context: WebhookContext) => {
 			await GithubWebhookMiddleware(deleteBranchWebhookHandler)(context);
 			break;
 		case "repository":
-			if (context.action === "deleted") {
```

Comment:
I think we should have both `created` and `deleted` repository action handlers here to have a common pattern.

Reply:
We still have `created` and `deleted` conditional checks, I have just moved them into `repositoryWebhookHandler`. I reckon its more relevent there and it tidies this function up a bit 🤷

Reply:
Just my two cents, I am happy to leave this to @joshkay10 your decision. Seems /nit to me. What @Harminder84 said has his point, however we already have bellow patterns:
```
case "issue_comment":
			if (context.action === "created" || context.action === "edited") {
				await webhookTimeout(GithubWebhookMiddleware(issueCommentWebhookHandler))(context);
			}
			break;
```

Reply:
Yeah it's a bit messy, we have a combination of logic at this level and then down a level on the handlers themselves. My opinion/thinking is it was more approriate for the condition to exist within the handler, I reckon this function should only have that top level decision of what handler, then the handler can decide the more finer details